Nclient.html

```diff
@@ -23,56 +23,435 @@
+      						<option value="21">MA</option>
+      						<option value="22">MI</option>
+      						<option value="23">MN</option>
+      						<option value="24">MS</option>
+      						<option value="25">MO</option>
+      						<option value="26">MT</option>
+      						<option value="27">NE</option>
+      						<option value="28">NV</option>
+      						<option value="29">NH</option>
+      						<option value="30">NJ</option>
+      						<option value="31">NM</option>
+      						<option value="32">NY</option>
+      						<option value="33">NC</option>
+      						<option value="34">ND</option>
+      						<option value="35">OH</option>
+      						<option value="36">OK</option>
+      						<option value="37">OR</option>
+      						<option value="38">PA</option>
+      						<option value="39">RI</option>
+      						<option value="40">SC</option>
+      						<option value="41">SD</option>
+      						<option value="42">TN</option>
+      						<option value="43">TX</option>
+      						<option value="44">UT</option>
+      						<option value="45">VT</option>
+      						<option value="46">VA</option>
+      						<option value="47">WA</option>
+      						<option value="48">WV</option>
+      						<option value="49">WI</option>
+      						<option value="50">WY</option>
+    						</select>
+    						<label>State</label>
+  						</div>
+						
+						<div class="input-field col s3">
+							<input type="text" name="zip_code">
+							<label for="zip_code">Zip Code</label>	
+						</div>
+						<div class="input-field col s5">
+							<input type="tel" name="other">
+							<label for="other">Other Number</label>
+						</div>
+						</div>
+
+
+					</div>
+				</form>
+			</div>
+		</div>
+
+
+		<div class="defendant">
+			<div class="container">
+				<h5>Defendant Information</h5>
+				<form action="" id="">
+					
+					<div class="row input-field col s5">
+						<input type="text" name="def_count">
+						<label for="def_count">How many defendants are there?</label>
+					</div>
+
+					
+					<div class="row">
+						<div class="input-field col s5">
+							<input type="text" name="def_first">
+							<label for="def_first">First Name</label>
+						</div>
+						<div class="input-field col s5">
+							<input type="text" name="def_last">
+							<label for="def_last">Last Name</label>							
+						</div>
+						<div class="input-field col s2">
+							<input type="text" name="mi">
+							<label for="mi">MI</label>
+						</div>
+					</div>
+
+					<div class="row">
+						<div class="input-field col s8">
+							<input type="text" name="def_add">
+							<label for="def_add">Address</label>
+						</div>
+					</div>
+
+					<div class="row">
+						<div class="input-field col s3">
+							<input type="text" name="def_city">
+							<label for="def_city">City</label>
 						</div>
+						<div class="input-field col s2">
+							<select>
+      						<option value="" disabled selected></option>
```

Comment:
remove `disabled selected` here, as the default will always be the first option unless you change that with javascript